misc/e2etest.py

```diff
@@ -150,9 +150,9 @@ def main():
     else:
         psqlstring = args.connection_string
 
-    xrun(['cmd/algorand-indexer/algorand-indexer', 'import', '-P', psqlstring, os.path.join(e2edata, 'blocktars', '*'), '--genesis', os.path.join(e2edata, 'algod', 'genesis.json')], timeout=20)
+    xrun(['algorand-indexer', 'import', '-P', psqlstring, os.path.join(e2edata, 'blocktars', '*'), '--genesis', os.path.join(e2edata, 'algod', 'genesis.json')], timeout=20)
```

Comment:
This now relies on `go install` or otherwise getting the binary into the PATH?

Reply:
Yes.  We can talk about this if you'd like to determine the best way to test this.

Reply:
these changes should be replaced by #120